src/lib/apirequest.ts

```diff
@@ -190,6 +190,10 @@ export function createAPIRequest<T>(
 
   options.headers = headers;
   options.params = params;
+  // We need to set a default content size, or the max defaults
+  // to 10MB.  Setting to 2GB by default.
+  // https://github.com/google/google-api-nodejs-client/issues/991
+  options.maxContentLength = options.maxContentLength || Math.pow(2, 31);
```

Comment:
Is it necessary to increase the default max? In my opinion that is the decision to be made by the application. Isn't it good enough to just allow the app developers to configure it, which is already supported?

I don't know but 2GB seems too high to me as the default value.

Reply:
The choice of 10MB was a somewhat random decision on the part of a transitive dependency we use.  This was not a conscious choice by us, or Axios. The change is very much semver major, and frankly - not something I think is necessary.  Moving to 2GB was my way of saying 'nah, this is fine'.  For the rare developer that wants to upload more than that... they have the option :P

Reply:
OK. I don't feel strongly about the default max anyway. Just one nit. Can you define a constant instead of inlining `Math.pow(2, 31)` here?